src/Hive.IO/Hive.IO/Results/Results.cs

```diff
@@ -670,11 +670,12 @@ public static double[] GetTotalMonthlySystemLossesNonRenewable(List<ConversionTe
             {
                 for (int i = 0; i < result.Length; i++)
                 {
-                    if (tech is SurfaceBasedTech == false)
+                    // only for fuel based and grid electricity. not for Heatpumps, because electricity from heatpumps will be attributed to consumed electricity (Outgoing arrows in Sankey)
+                    if (tech is SurfaceBasedTech == false && tech is HeatPump == false)
```

Comment:
`if (!(tech is SurfaceBasedTech || tech is HeatPump))` ? hmm... in Python we use `not`, which is much easier to read than `!`... I'm not sure it's better here so... let's leave this!